newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -81,30 +81,26 @@ type ClosableTabsProps = {|
   children: Node,
 |};
 
-export class ClosableTabs extends Component<ClosableTabsProps> {
-  render() {
-    const { hideLabels, children } = this.props;
-
-    return (
-      <ThemeConsumer>
-        {muiTheme => {
-          const tabItemContainerStyle = {
-            maxWidth: '100%', // Tabs should take all width
-            flexShrink: 0, // Tabs height should never be reduced
-            display: hideLabels ? 'none' : 'flex',
-            flexWrap: 'nowrap', // Single line of tab...
-            overflowX: 'auto', // ...scroll horizontally if needed
-            backgroundColor: muiTheme.closableTabs.containerBackgroundColor,
-          };
+export function ClosableTabs({ hideLabels, children }: ClosableTabsProps) {
+  return (
```

Comment:
That can be reduced to `export const ClosableTabs = ({ hideLabels, children }: ClosableTabsProps) => (` and remove the return.